Added getSubscription method

patelhir opened the pull request.

ricbra commented:
Please rebase your commits. It seems you deleted and readded service.php?
Also I see the same commit message twice.

Could you also add tests for the new call?

Op di 30 aug. 2016 06:57 schreef patelhir <notifications@github.com>:

> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/netvlies/basecamp-php/pull/37
> Commit Summary
>
>    - Add files via upload
>    - Delete service.php
>    - Add files via upload
>
> File Changes
>
>    - *M* src/Basecamp/Resources/service.php
>    <https://github.com/netvlies/basecamp-php/pull/37/files#diff-0> (1881)
>
> Patch Links:
>
>    - https://github.com/netvlies/basecamp-php/pull/37.patch
>    - https://github.com/netvlies/basecamp-php/pull/37.diff
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/netvlies/basecamp-php/pull/37>, or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA1Rts-NN1BYG-0b3IbNHGn8dItZ2aRJks5qk7gjgaJpZM4JwJU4>
> .
>

patelhir commented:
By mistake I added service.php into root folder, so I deleted it and reupload again on proper place. 

I am new to GIT and I don't know how to rebase.
